src/Tribe/Aggregator/Event.php

```diff
@@ -349,27 +353,27 @@ public static function preserve_changed_fields( $data ) {
 				continue;
 			}
 
-			// if we don't have a field to reset to, let's unset the event meta field
-			if ( ! isset( $post_meta[ $field ] ) ) {
-				unset( $data[ $field ] );
```

Comment:
Don't we need to still unset the data field? I mean from what I remember without it would break.